firmware/boron-ins-fsm/test/mocks/mock_ble.h

```diff
@@ -4,89 +4,76 @@
 
 #pragma once
 
-#define BLE_GAP_ADV_SET_DATA_SIZE_MAX   (31)   //< Maximum data length for an advertising set.
-#define BLE_MAX_ADV_DATA_LEN            BLE_GAP_ADV_SET_DATA_SIZE_MAX
+#define BLE_GAP_ADV_SET_DATA_SIZE_MAX (31)  //< Maximum data length for an advertising set.
+#define BLE_MAX_ADV_DATA_LEN          BLE_GAP_ADV_SET_DATA_SIZE_MAX
 
 #include "../inc/spark_wiring_vector.h"
 
 // Defines fake BleAdvertisingDataType values
-enum class BleAdvertisingDataType : uint8_t {
-    MANUFACTURER_SPECIFIC_DATA
-};
+enum class BleAdvertisingDataType : uint8_t { MANUFACTURER_SPECIFIC_DATA };
 
 // Fake class for BleAddress object
 class BleAddress {
-public:
+   public:
```

Comment:
Not sure what I think about these 3 spaces either. Was the old code consistently putting the `public`, `private`, `protected` labels in line with `class`? If so, it would be nice to keep it that way.

Reply:
Should be fixed after I changed AccessModifierOffset to -4.